test/test_foreach.py

```diff
@@ -302,9 +306,12 @@ def test_pointwise_op_fastpath(self, device, dtype, op):
             self._test_pointwise_op(device, dtype, op, N, True, disable_fastpath)
             for scalar in Scalars:
                 self._test_pointwise_op(device, dtype, op, N, True, disable_fastpath, values=scalar)
-            for _, scalarlist in getScalarLists(N):
+                self._test_pointwise_op(device, dtype, op, N, True, disable_fastpath, values=torch.tensor(scalar))
```

Comment:
Should you also test the case where `torch.tensor(scalar, device=device)` ?
Same for all the updates below.

Reply:
Yes, we'll need to add more "assertRegexFail" type tests